src/rules/padding-around-test-blocks.ts

```diff
@@ -27,6 +27,6 @@ export const config = [
 
 export default createPaddingRule(
   __filename,
-  'Enforce padding around afterAll blocks',
+  'Enforce padding around `test` blocks',
```

Comment:
```suggestion
  'Enforce padding around `test` and `it` blocks',
```

Reply:
I fixed in [5f7c7b0](https://github.com/jest-community/eslint-plugin-jest/pull/1691/commits/5f7c7b02267d738dcd047a4793b0645447a25aec).